src/Fhir.Anonymizer.Core/Processors/RedactProcessor.cs

```diff
@@ -51,7 +51,7 @@ public void Process(ElementNode node)
             }
             else
             {
-                node.Value = string.Empty;
+                node.Value = null;
```

Comment:
So we are going to remove all redacted fields in output. What if a user wants to keep the empty field?

Reply:
Got it. According to the latest discussion in working item #72757, removing the fields will be the only supported behavior.

Reply:
Got it!